src/arm/radxa_rock3c.c

```diff
@@ -22,7 +22,7 @@
 const char* radxa_rock3c_serialdev[MRAA_RADXA_ROCK3C_UART_COUNT] = { "/dev/ttyS3", "/dev/ttyS4", "/dev/ttyS5", "/dev/ttyS9" };
 
 void
-mraa_radxa_rock3c_pininfo(mraa_board_t* board, int index, int sysfs_pin, mraa_pincapabilities_t pincapabilities_t, char* fmt, ...)
+mraa_radxa_rock3c_pininfo(mraa_board_t* board, int index, int gpio_chip, int gpio_line, mraa_pincapabilities_t pincapabilities_t, char* fmt, ...)
```

Comment:
```suggestion
mraa_radxa_rock3c_pininfo(mraa_board_t* board, int index, int gpio_chip, int gpio_line, mraa_pincapabilities_t pincapabilities_t, char* pin_name)
```
1. 我们没有用va_args
2. 我们没有fmt,直接提供了pin的名字